pom.xml

```diff
@@ -337,6 +338,12 @@
                 <groupId>com.facebook.presto</groupId>
                 <artifactId>presto-hive</artifactId>
                 <version>${project.version}</version>
+                <exclusions>
```

Comment:
this exclusion is probably not needed. I tried building without this and the build went fine.